docs/reference/config-options.md

```diff
@@ -48,3 +49,14 @@ IMPORTANT: If you switch this back to `disabled-permanently` from
 `disabled-by-default`, LegalHold devices may still be active in teams
 that have created them while it was allowed.  This may change in the
 future.
+
+### Team Search Visibility
+
+Is a team allowed to change its team search visibility settings?  If a
+team is enabled, it can be configured so other teams are completely
+invisible for search.
```

Comment:
```suggestion
Is a team allowed to change its team search visibility settings? If enabled
for the team, it can be configured so that non team users do not show up in search.
```